tests/platforms/kubernetes/kubeobj.go

```diff
@@ -107,7 +107,21 @@ func buildDaprAnnotations(appDesc AppDescription) map[string]string {
 	if appDesc.DaprEnv != "" {
 		annotationObject["dapr.io/env"] = appDesc.DaprEnv
 	}
-
+	if appDesc.EnableAppHealthCheck {
+		annotationObject["dapr.io/enable-app-health-check"] = "true"
```

Comment:
I think we need a preview feature flag for user's awareness that we can change this behavior still.